### DISCUSSION TITLE
Hedy for university students (Hedy "senior"?)

### DISCUSSION THREAD

**Discussion by Felienne** [hedyorg/hedy#4040] [Large Projects & Plans]
First discussed in https://github.com/hedyorg/hedy/discussions/4036#discussioncomment-4987237

What could we do to make Hedy more interesting for older students (> 16)?

I think that this could just be a matter of content. If we make some adventures that are more tailored towards seriousness, the trajectory I think is good for any age, and of course older people can go through it a bit more quickly.

The one feature that I think could be cool is to add the option to upload small data files (csv, json maybe) that you could analyse as a means to make larger and more challenging assignments (of shich some would even be col for younger kids)

But!! I am mainly opening this discussion to say we are not going to work on it. There are many reasons for this but the main one is that we have work enough as is :) 

However, that I am totally open to making some (small) tweaks to help others to work on it! If someone wants to pull it by authoring content, we can easily ask teachers on signup if they want the default adventure set or a 16+ one. So short story even shorter: the first step if someone wants to pursue this is to start authoring some content in the adventures and tell us here!

**Comment by ogoletti** [comment]
Caveat: please, make it so that "seriousness" != "math". Maybe getting inspiration from media-computation for the context would make sense for this public ?

Also, maybe it would make sense to slowly transition towards "english only" for this public?

**Reply by Felienne** [reply]
Yeah that is a good point @ogoletti! 

I personally think the turtle adventures we have would also be lovely for students (esp non-CS) but if are catering to programming professors (esp CS professors) we will need to cater to what they think is serious (one reason that this is less interesting for me to work on)

**Reply by feefladder** [reply]
Within the Rust community, there is discussion going on for internationalizing the language, and there's [koro](https://github.com/ChimeraCoder/koro): Go in Bengalese. It's outside the scope of this project, but I hope that in the future, there is no need to transition to English-only.

**Reply by Felienne** [reply]
I am not sure this is outside the scope of this project! We too hope for a future with more localized programming languages!!

**Comment by jpelay** [comment]
I think it's totally ok for us as a team not to create content for this demographic, as it isn't our priority right now. Besides, we already give teachers the freedom to customize and create their own adventures, so it's totally possible for them to use the Hedy programming language and not the Hedy content (after all, if they're teaching with C, Python etc no one expects their creators and maintainers to create official content to use as teaching assistance). So perhaps at this level maybe the distinction between the Hedy teaching platform and the Hedy programming language is worthwhile.

In this same note, perhaps we can add a way for teachers to create their own quizzes? Maybe their own parsons?

**Reply by Felienne** [reply]
Yeah we have considered parsons and quizzes by teachers but we never put in the energy, because it is kind of complex. Now that we are doing a run of teacher feedback, let's see if this is a point they want!
